x-pack/plugins/index_management/server/routes/api/mapping/index.ts

```diff
@@ -6,3 +6,4 @@
  */
 
 export { registerMappingRoute } from './register_mapping_route';
+export { registerUpdateMappingRoute } from './register_update_mapping_route';
```

Comment:
nit: these 2 exports should be replaced with `registerIndexMappingRoutes`

Reply:
added in https://github.com/elastic/kibana/pull/178923/commits/fb98148ae91566ce3461bf4ba024624f44270cdb